### PR TITLE
jamie23's Intellij Goodie

### PR DESCRIPTION
**Pull request by jamie23**
**What does your Instant Answer do?**
My answer displays useful shortcuts for the IntelliJ IDEA software.

**What is the data source for your Instant Answer? (Provide a link if possible)**
Shortcut world:
http://www.shortcutworld.com/en/win/IntelliJ.html

**Why did you choose this data source?**
I've seen it recommended for the eclipse cheat sheet, having read over the IntelliJ commands, all the shortcuts I use are there and a bunch which I've since learnt from reading it.

**Are there any other alternative (better) data sources?**
Not that I have came across.

**Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)**
Developers, geeks, programmers. 

**Is this Instant Answer connected to a DuckDuckHack [Instant Answer idea](https://duck.co/ideas)?**
Yes, 
https://duck.co/ideas/idea/6324/keyboard-shortcuts-cheatsheet-for-intellij-idea

**Are you having any problems? Do you need our help with anything?**
Not so far.

**Where did you hear about DuckDuckHack? (For first time contributors)**
Began to use DDG and wanted to contribute.

**What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)**
screen shot 2015-09-01 at 00 30 20

-----
IA Page: https://duck.co/ia/view/intellij_cheat_sheet

### CODE REVIEW

File: share/goodie/cheat_sheets/json/intellij.json [hunk 1]
```diff
@@ -0,0 +1,261 @@
...
+            "key": "[Ctrl] [`]",
+            "val": "Show/Hide Console"
+        }, {
+            "key": "[Shift] [Shift]",
+            "val": "Search everywhere"
+        }, {
+            "key": "[Ctrl]  [F]",
+            "val": "Find"
+        }, {
+            "key": "[F3]",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/intellij.json]: You don't actually want/need the `[]` here. They're only necessary for `key`'s with spaces to indicate what part is the keyboard shortcut

Comment (jamie23) [inline share/goodie/cheat_sheets/json/intellij.json]: Ah sorry about that, thanks for letting me know, should be cleaned up now.

**Comment by moollaza** [comment]
@jamie23 this LGTM :+1:

Thanks a lot for contributing (we really appreciate it :smiley:). We invite you to stick around and comment/help on other PR's & Issues or even submit an Instant Answer!

**Comment by jamie23** [comment]
@moollaza Thanks man! Was a lot of fun working with the tools, really great job on those!

**Comment by moollaza** [comment]
@jamie23 congrats! It's live: https://duckduckgo.com/?q=intellij+cheat+sheet&ia=cheatsheet

Thanks again for taking the time to contribute! We really appreciate it. Feel free to stick around and comment/help on other PR's, tackle open issues or even submit more Instant Answers!

**Comment by moollaza** [comment]
@jamie23 By the way, we think you did a great job with this! Are you interested in helping others who build similar Instant Answers in the future?

If yes, we'll reach out and add you to a private list of mentors as well as send over some DDG merch as an additional thank you. Just email <open@duckduckgo.com> with the subject 'I want to help.' Please include your shipping address, phone-number (required for shipping outside USA) and t-shirt size. We'll reach out from there!

Otherwise, thank you for this amazing contribution. We hope you'll keep exploring and playing around with DuckDuckHack.

**Pull request merged**